Documentation/doc/scripts/test_doxygen_versions.sh

```diff
@@ -126,6 +128,7 @@ if [ "$HAS_REF" -ne "1" ]; then
     CGAL_NAME="$(cat $ROOT/VERSION)"
     cd $ROOT/doc/scripts
   fi
+  echo "CGAL_NAME = $CGAL_NAME"
```

Comment:
This line seems a leftover of a debug session.

Reply:
Indeed